task/backend/analytical_storage.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/influxdata/influxdb/v2/kit/errors"
```

Comment:
TIL :) i was thinking the one from stdlib, but this works great.

Reply:
I actually had no idea I imported that.. I thought I was using the stdlib 😄